src/app/utils/withAuth/index.ts

```
@@ -0,0 +1,51 @@
+      const authHeader = headersList.get('Authorization');
+
+      if (!authHeader?.startsWith('Bearer ')) {
+        return NextResponse.json(
+          { message: 'unauthorized' },
+          { status: 401 }
+        );
+      }
+
+      const token = authHeader.split(' ')[1];
+      const secret = new TextEncoder().encode(process.env.NEXTAUTH_SECRET!);
+        console.log(token);
```

ESLint

[error] Unexpected console statement. Only these console methods are allowed: warn, error.

(no-console)